dogstatsd.py

```diff
@@ -517,7 +529,7 @@ def init(config_path=None, use_watchdog=False, use_forwarder=False, args=None):
     if non_local_traffic:
         server_host = '0.0.0.0'
 
-    server = Server(aggregator, server_host, port, forward_to_host=forward_to_host, forward_to_port=forward_to_port)
+    server = Server(aggregator, server_host, port,  buffer_size=buffer_size, forward_to_host=forward_to_host, forward_to_port=forward_to_port)
```

Comment:
nit: this is probably too long as well... I usually _try_ to respect the prototype (even though python doesn't really care with named parameters, so this is by all means correct), so I'd set buffer_size at the end too. But whatever, that's just me feel free to ignore that part.